resources/assets/js/app.js

```diff
@@ -22,7 +22,7 @@ var App = Vue.extend({
         };
     },
     created: function () {
-        this.$http.get('topics', function (data, status, request) {
+        this.$http.get('topics?status=accepted', function (data, status, request) {
```

Comment:
Nope, we actually want to list out all proposed topics for the purpose of voting.